std_msgs/ColorRGBA color1    # RGB values [0-1] for region 1
std_msgs/ColorRGBA color2    # RGB values [0-1] for region 2
std_msgs/ColorRGBA color3    # RGB values [0-1] for region 3
std_msgs/ColorRGBA color4    # RGB values [0-1] for region 4
int32  blinks                # Number of blinks (1 blink = 1 time on + 1 time off)
float32 duration_on          # in [s]
float32 duration_off         # in [s]
int32 start_led             # First LED in the stripe where LEDs will blink
int32 end_led               # Last LED in the stripe where LEDs will blink
---
int32 blinks_left           # Number of blinks left until the goal is completed (change this to some decent feedback)
---
int32 blinks_left           # Number of blinks left until the goal is completed
